backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -50,65 +63,128 @@ private void updatePersonFacility(Person person, UUID facilityId) {
     person.setFacility(facility);
   }
 
-  @AuthorizationConfiguration.RequirePermissionSearchPatients
-  public List<Person> getPatients(UUID facilityId, int pageOffset, int pageSize) {
-    return _repo
-        .findByFacilityAndOrganization(
-            _os.getFacilityInCurrentOrg(facilityId),
-            _os.getCurrentOrganization(),
-            false,
-            PageRequest.of(pageOffset, pageSize, NAME_SORT))
-        .toList();
+  // For readablility below. Get the OrgId for the currently logged in user.
+  // don't trust an id provided by the user.
+  private UUID getOrgId() {
+    return _os.getCurrentOrganization().getInternalId();
   }
 
-  @AuthorizationConfiguration.RequirePermissionSearchPatients
-  public long getPatientsCount(UUID facilityId) {
-    return _repo.countAllByFacilityAndOrganization(
-        _os.getFacilityInCurrentOrg(facilityId), _os.getCurrentOrganization(), false);
+  // Specifications filters for queries
+  private Specification<Person> inWholeOrganizationFilter() {
+    return (root, query, cb) ->
+        cb.equal(root.get(SpecField.Organization).get(SpecField.InternalId), getOrgId());
```

Comment:
Should be able to make this `cb.equal(root.get(SpecField.Organization), os.getCurrentOrganization());` Or is there a catch?

Reply:
I was using UUID for facility since that's what the graphql passed in, so I continued that logic on to Org. Your suggestion works and is easier to read.